seacrowd/sea_datasets/seahorse/seahorse.py

```diff
@@ -0,0 +1,195 @@
+from pathlib import Path
+
+import datasets
+import pandas as pd
+import tensorflow_datasets as tfds
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@inproceedings{clark-etal-2023-seahorse,
+    title = "{SEAHORSE}: A Multilingual, Multifaceted Dataset for Summarization Evaluation",
+    author = "Clark, Elizabeth  and
+      Rijhwani, Shruti  and
+      Gehrmann, Sebastian  and
+      Maynez, Joshua  and
+      Aharoni, Roee  and
+      Nikolaev, Vitaly  and
+      Sellam, Thibault  and
+      Siddhant, Aditya  and
+      Das, Dipanjan  and
+      Parikh, Ankur",
+    editor = "Bouamor, Houda  and
+      Pino, Juan  and
+      Bali, Kalika",
+    booktitle = "Proceedings of the 2023 Conference on Empirical Methods in Natural Language Processing",
+    month = dec,
+    year = "2023",
+    address = "Singapore",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2023.emnlp-main.584",
+    doi = "10.18653/v1/2023.emnlp-main.584",
+    pages = "9397--9413",
+}
+"""
+
+_DATASETNAME = "seahorse"
+
+_DESCRIPTION = """
+SEAHORSE is a dataset for multilingual, multifaceted summarization evaluation. It consists of 96K summaries with human
+ratings along 6 quality dimensions: comprehensibility, repetition, grammar, attribution, main idea(s), and conciseness,
+covering 6 languages, 9 systems and 4 datasets.
+"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/seahorse"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = "https://storage.googleapis.com/seahorse-public/seahorse_data.zip"
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+# The original dataset only contaions gem_id, we need to retrieve the article following https://github.com/google-research-datasets/seahorse?tab=readme-ov-file#retrieving-articles-from-gem
+def get_wikilingual_data(lang, split):
+    ds, info = tfds.load(f"huggingface:gem/wiki_lingua_{lang}", split=split, with_info=True)
```

Comment:
We also tried to look for alternatives other than using `tfds`, but couldn't find them. We also found that the wiki_lingua in HF and `tfds` are different in numbers. It could be that the hf dataset is GEMv2 (not GEMv1) and has been re-split. To ensure reproducibility, it would be better to proceed with `tfds` and modify the dataloader to require `tfds` with try-catch, @chenxwh.

Example:
```
try:
    import yourModule
except ImportError:
    raise Exception(“…”)
```

cc: @fhudi @SamuelCahyawijaya @sabilmakbar

Reply:
@holylovenia thanks for the comment, you mean do not add `tensorflow-dataset` in the requirements.txt either? but try import `tfds` in line 65? 
And `tensorflow`? we don't need to import it explicitly in the script but it is required by `tfds`

Reply:
Ah yeah, no need to add the additional packages that you need for this specific dataloader, e.g., `tensorflow`, to the requirements.txt. We can just use try-catch to ensure the user has it if they specifically want to use this dataloader.

Reply:
Thanks @holylovenia, updated now, put more general exception to catch the error.